commit e170b512068f8c79a33291e16f3b5691b4aced34
Author: Albert Astals Cid <aacid@kde.org>
Date:   Fri Sep 28 23:04:18 2012 +0200

    Removing installed headers between 4.9.1 and 4.9.2 is not a good idea
    
    Partly reverts 386532a02774eecb716db8a87dc72465a1cbee29 by Christoph Feck
    Fixes gwenview build
    CCMAIL: christoph@maxiom.de

diff --git a/includes/CMakeLists.txt b/includes/CMakeLists.txt
index f1c843b..342ff7e 100644
--- a/includes/CMakeLists.txt
+++ b/includes/CMakeLists.txt
@@ -960,6 +960,8 @@ DESTINATION ${INCLUDE_INSTALL_DIR}/KDE/khtml COMPONENT Devel)
 
 install(FILES
   Nepomuk/File
+  Nepomuk/KRatingPainter
+  Nepomuk/KRatingWidget
   Nepomuk/MassUpdateJob
   Nepomuk/Resource
   Nepomuk/ResourceManager
diff --git a/includes/Nepomuk/KRatingPainter b/includes/Nepomuk/KRatingPainter
new file mode 100644
index 0000000..300dd7d
--- /dev/null
+++ b/includes/Nepomuk/KRatingPainter
@@ -0,0 +1 @@
+#include "../../nepomuk/kratingpainter.h"
diff --git a/includes/Nepomuk/KRatingWidget b/includes/Nepomuk/KRatingWidget
new file mode 100644
index 0000000..c76b4ea
--- /dev/null
+++ b/includes/Nepomuk/KRatingWidget
@@ -0,0 +1 @@
+#include "../../nepomuk/kratingwidget.h"
